src/components/WalletAccounts.vue

```diff
@@ -71,10 +71,13 @@
         </ListItem>
         <div class="account-assets" :class="{ active: shouldExpandAccount(account) }">
           <ListItem
-            v-if="account.nfts && account.nfts.length > 0 && !isAssetList"
+            v-if="account.nfts && !isAssetList"
             @item-selected="
               $router.push({
-                path: `/wallet/nfts/activity/${account.id}`
+                path: `/wallet/nfts/activity/${account.id}`,
+                query: {
+                  tab: getDefaultTab(account)
```

Comment:
@mpwanyi256 why are we adding this extra query? I don't see where it's being used

Reply:
<img width="717" alt="Screenshot 2022-11-08 at 14 48 12" src="https://user-images.githubusercontent.com/20843520/200556411-b604d049-255e-4ceb-9c66-9822c18863c9.png">

@Jennievon We needed to direct the user with 0 NFT to the activity tab. So inside the `src/views/Wallet/NFTActivity.vue` component, there is already a check for that, I just needed to add a query with a default tab to switch the user to